proteus/mprans/MCorr.h

```diff
@@ -69,6 +69,56 @@ namespace proteus
 				   int* exteriorElementBoundariesArray,
 				   int* elementBoundaryElementsArray,
 				   int* elementBoundaryLocalElementBoundariesArray)=0;
+    virtual void calculateResidual_explicit_comp_quad(//element
```

Comment:
Don't copy-paste a new residual function for this. Just add a switch (e.g. MOD_MCORR) and add only what you need. Copy-pasting like this is generally a bad idea for a small change, because it  makes it very difficult to see what you've changed.

Reply:
Can you delete this declaration now that the changes are applied only to calculateResidual?